osx/KA-Lite/KA-Lite/README.md

```diff
@@ -6,20 +6,20 @@ This is the Xcode project for the KA Lite application.
 
 ## Requirements
 
-Xcode 6.x or 7.x
+Xcode 7.2
```

Comment:
Xcode 7.2 or higher?  Since macOS Sierra I think has Xcode 8.x already and we've tested this works on that too.